DASH-IF-IOP.bs.md

```diff
@@ -10,8 +10,500 @@ Live discussion in #document-authoring on Slack.
+
+## XLink ## {#timing-xlink}
+
+Issue: Determine appropriate content for this section.
+
+## Leap seconds ## {#timing-leapseconds}
+
+This section is intentionally left blank to indicate that the leap seconds topic is out of scope of this proposal.
+
+## Forbidden techniques ## {#timing-nonos}
+
+Some aspects of [[!MPEGDASH]] are not compatible with the interoperable timing model defined in this document. In the interest of clarity, they are explicitly listed here:
+
+* The `@presentationDuration` attribute SHALL NOT be used.
+
+## Dynamic MPD updates ## {#timing-mpd-updates}
+
+Dynamic MPDs MAY change over time. The nature of the change is not restricted unless such a restriction is explicitly defined.
+
+Some common reasons to make changes in dynamic [=MPDs=]:
+
+* Adding new [=media segments=] to an existing [=period=].
+* Adding new [=periods=].
+* Adjusting the durations and start times of [=periods=].
+* Removing [=media segments=] and/or [=periods=] that have fallen out of the time shift window.
+* Adding `MPD@mediaPresentationDuration` and (on the last [=period=]) `Period@duration` to signal that a live service has ended.
+* Converting the [=MPD=] to a static [=MPD=] to signal that a live service has become available on-demand as a recording.
+
+The following restrictions are defined here for MPD updates:
+
+* `MPD@id` SHALL NOT change.
+* `MPD@availabilityStartTime` SHALL NOT change.
+* `Period@id` SHALL NOT change.
+* `Period@start` SHALL NOT change.
+* The adaptation sets present in a [=period=] (i.e. the set of `AdaptationSet@id` values) SHALL NOT change.
+* The functional behavior of a representation (identified by a matching `Representation@id` value) shall not change, neither in terms of metadata-driven behavior (including metadata inherited from adaptation set level) nor in terms of segment timing.
+
+Issue: Do we need to restrict to "extend-only" behavior? https://github.com/Dash-Industry-Forum/DASH-IF-IOP/issues/224
+
+Issue: Review synchronization with 4.4.3.3
+
+Additional restrictions can be defined by other parts of this document.
+
+Clients SHOULD use `Period@id` to track [=period=] identity across [=MPD=] updates.
+
+The presence of `MPD@minimumUpdatePeriod` SHALL be used to signal that the [=MPD=] is expected to change. The value of this field indicates the frequency at which a client is expected to poll for [=MPD=] updates.
+
+Note: In other words, `MPD@minimumUpdatePeriod` is the maximum expected delay between an update being made to the [=MPD=] on the service side and this update becoming available to a client. For practical considerations, you should add on top of this the time spent on acquiring and processing the updated [=MPD=].
+
+If `MPD@minimumUpdatePeriod` is absent then `MPD@mediaPresentationDuration` SHALL be present.
+
+Issue: Is it OK to have the two be present simultaneously? That is, a fixed-size MPD that may still change? Does it match any IOP use cases? I think not but have to double-check.
```

Comment:
Note: This relates to a comment I've posted further up. I think it makes sense to allow this. Possibly to allow removal from the start of the live window, but also because it allows flexibility for manifest updates when it's known no further content will be appended, which may be useful for cases we've not considered?

Reply:
Agreed.